ci/github/unit-testing/dynamic-sidecar.bash

```diff
@@ -20,7 +20,7 @@ test() {
   # shellcheck source=/dev/null
   source .venv/bin/activate
   pushd services/dynamic-sidecar
-  make test-ci-unit
+  make test-ci-unit pytest-parameters="--log-cli-level=DEBUG"
```

Comment:
We had this in the old sidecar and was far too many logs ... just keep an eye!

Reply:
reverted this, was trying to figure out an issue